### PR TITLE
Monitoring (Alerting and Metrics) content addition

### PR DESCRIPTION
**Pull request by jeanchlopez**
This new section includes details about the built-in alerting, monitoring and dashboarding features of OCP/OCS. Based on our discussions tomorrow or later we will decide if we make an extra addition detailing the Grafana workaround that was created using dittybopper.

The idea is to provide sufficient information but to avoid leading all attendees to massively deploy local Grafana instances an open CEE tickets when they will make it on-site in my opinion.

### CODE REVIEW

File: ocp4ocs4/ocs4.adoc [hunk 1]
```diff
@@ -1606,6 +1606,138 @@ This is an effective way to scale out capacity and performance if you observe th
...
+image::imgs/metrics-alertinghomepage.png[OCP Alerting Homepage]
+
+You can display the alerts in the main window by state. To do so you must highlight the states you want to display. The states are:
+
+* `Firing` - Alert has been confirmed
+* `Silenced` - Alerts that have been silenced while they were in `Pending` or `Firing` state
+* `Pending` - Alerts that have been triggered but not confirmed
+* `Not Firing` - Alerts that have not been triggered
+
+NOTE: An alert transitions from `Pending` to `Firing` state if it persists for more than 10 minutes.
```
Comment (mulbc) [inline ocp4ocs4/ocs4.adoc]: That's not true... most alerts are configured with a 10 min delay, but we also have some with 3 mins or 5 mins... This is configurable for every single alert

File: ocp4ocs4/ocs4.adoc [hunk 2]
```diff
@@ -1606,6 +1606,138 @@ This is an effective way to scale out capacity and performance if you observe th
...
+
+.OCP Metrics Menu
+image::imgs/metrics-metricsleftpanemenu.png[OCP Metrics Menu]
+
+This will take you to the *Metrics* homepage as illustrated below.
+
+.OCP UI Metrics Homepage
+image::imgs/metrics-queryfield.png[OCP Monitoring Metrics Homepage]
+
+Use the query field to either enter the formula of your choice or to search for metrics by name. The metrics available will let you query both OCP related information or Ceph related information. The queries can be simple or complex using the Prometheus query syntax and all its available functions.
```
Comment (mulbc) [inline ocp4ocs4/ocs4.adoc]: `both OCP related information or Ceph related information`
let's rephrase this to:
`both OCP related information or OCS related information` - because you can also get Noobaa information there...

File: ocp4ocs4/ocs4.adoc [hunk 3]
```diff
@@ -1606,6 +1606,138 @@ This is an effective way to scale out capacity and performance if you observe th
...
+.Grafana Homepage
+image::imgs/metrics-grafanahomepage.png[Grafana Homepage]
+
+Click the *Home* menu in the top left corner to choose one of the packaged OCP dashboards.
+
+.Grafana OCP Dashboard
+image::imgs/metrics-ocpdashboard.png[Grafana Dashboard]
+
+The illustration above is a screen capture of the `Kubernetes / Compute Resources / Cluster` dashboard.
+
```
Comment (mulbc) [inline ocp4ocs4/ocs4.adoc]: I'd put a note at the end of this sub-section explaining that there are currently no OCS related dashboards in the built-in Grafana instance.

**Comment by jeanchlopez** [comment]
Try to address the remarks.
